bats/tests/pyrsia_node_maven_build_service.bats_

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# TODO This test is disabled for now, more info in https://github.com/pyrsia/pyrsia/issues/1140
```

Comment:
Does this comment actually disable the test? Or do we have to prepend `skip` or something like that (https://stackoverflow.com/questions/72127278/how-to-disable-or-ignore-a-test-in-bash-automated-testing-system)

Reply:
The bats test file is renamed so it's not picked up when the tests are executed.

Reply:
It's renamed because I don't want the file setup and test setup for this file to be executed (it takes time and resources and the test is not executed)

Reply:
got it.